met-web/src/components/publicDashboard/KPI/TotalResponses.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { RadialBarChart, PolarAngleAxis, RadialBar } from 'recharts';
+import Stack from '@mui/material/Stack';
+import { MetPaper, MetHeader3 } from 'components/common';
+import { Grid, Skeleton } from '@mui/material';
+import SupervisorAccountIcon from '@mui/icons-material/SupervisorAccount';
+import { DASHBOARD } from '../constants';
+import { ErrorBox } from '../ErrorBox';
+
+const HEIGHT = '213px';
+const TotalResponses = () => {
+    // Sample data
+    const sampleData = [{ name: 'L1', value: 100 }];
+    const circleSize = 150;
```

Comment:
just thinking out aloud....
was just wondering if there should be some indication that its a graph in the name..
ie instead of TotalResponses ;should we call it as TotalResponsesGraph or something of that sort?

Reply:
added Gauge to the name of these, as for the others "Trend" and "Bar" should cover it